pkg/generators/forwarding/fluentd/templates.go

```diff
@@ -904,9 +923,15 @@ const storeSyslogTemplate = `{{- define "storeSyslog" -}}
 	  message_key {{.PayloadKey}}
 	</format>
 {{end -}}
+{{- if .Hints.Has "pick_daemon_name" }}
+  <buffer $.systemd.u.SYSLOG_IDENTIFIER>
```

Comment:
I don't see a helper for "ChunkKeys" but we should add one

Reply:
Fixed it.